natgeo/mortar
# [SYTO-1143] Card Images now update consistently

To test:

On mortar run:
- `gulp build`
- `npm pack` This will create file natgeo-mortar_v3-3.1.7.tgz

On GN:
- Modify mortar_v3 entry on package.json:
`"@natgeo/mortar_v3": "file:(full path to the file generated above)/natgeo-mortar_v3-3.1.7.tgz"
- Run `npm install`
- Run `npm start`
- Verify that card images are updated consistently when hovering on the contextual menu.
